tests/default_vars.sh

```diff
@@ -23,13 +23,35 @@ elif [[ $MACHINE_ID = wcoss_dell_p3 ]]; then
   TASKS_stretch=48 ; TPN_stretch=28 ; INPES_stretch=2 ; JNPES_stretch=4
   TASKS_strnest=96 ; TPN_strnest=28 ; INPES_strnest=2 ; JNPES_strnest=4
 
+  TASKS_cpl_atmw=196; TPN_cpl_atmw=28; INPES_cpl_atmw=3; JNPES_cpl_atmw=8
+  THRD_cpl_atmw=1; WPG_cpl_atmw=6; APB_cpl_atmw="0 149"; WPB_cpl_atmw="150 195"
+
+  TASKS_cpl_atmw_gdas=560; TPN_cpl_atmw_gdas=14; INPES_cpl_atmw_gdas=6; JNPES_cpl_atmw_gdas=8
+  THRD_cpl_atmw_gdas=2; WPG_cpl_atmw_gdas=24; APB_cpl_atmw_gdas="0 311"; WPB_cpl_atmw_gdas="312 559"
+
   TASKS_cpl_dflt=192; TPN_cpl_dflt=28; INPES_cpl_dflt=3; JNPES_cpl_dflt=8
   THRD_cpl_dflt=1; WPG_cpl_dflt=6;  MPB_cpl_dflt="0 143"; APB_cpl_dflt="0 149"
   OPB_cpl_dflt="150 179"; IPB_cpl_dflt="180 191"
 
-  TASKS_cpl_dflt_wwav=212; TPN_cpl_dflt_wwav=28; INPES_cpl_dflt_wwav=3; JNPES_cpl_dflt_wwav=8
+  TASKS_cpl_dflt_wwav=196; TPN_cpl_dflt_wwav=28; INPES_cpl_dflt_wwav=3; JNPES_cpl_dflt_wwav=8
   THRD_cpl_dflt_wwav=1; WPG_cpl_dflt_wwav=6;  MPB_cpl_dflt_wwav="0 143"; APB_cpl_dflt_wwav="0 149"
-  OPB_cpl_dflt_wwav="150 179"; IPB_cpl_dflt_wwav="180 191";  WPB_cpl_dflt_wwav="192 211"
+  OPB_cpl_dflt_wwav="150 169"; IPB_cpl_dflt_wwav="170 177"; WPB_cpl_dflt_wwav="178 195"  
+  NPROC_ICE_cpl_dflt_wwav=8
+
+  TASKS_cpl_thrd_wwav=120; TPN_cpl_thrd_wwav=14; INPES_cpl_thrd_wwav=3; JNPES_cpl_thrd_wwav=4
+  THRD_cpl_thrd_wwav=2; WPG_cpl_thrd_wwav=6;  MPB_cpl_thrd_wwav="0 71"; APB_cpl_thrd_wwav="0 77"
+  OPB_cpl_thrd_wwav="78 97"; IPB_cpl_thrd_wwav="98 107"; WPB_cpl_thrd_wwav="108 119"
```

Comment:
OCN is still taking 20 tasks, does this mean OCN does not have threading turned on?

Reply:
@jiandewang can you comment on this question from Jun about MOM6 threading. 

@junwang-noaa there is about to be a fairly large update to this PR based on requests for changes, FYI.  Just doing the final testing now.

Reply:
@junwang-noaa threading in OCN won't gain speed, thus we are not using this

Reply:
@jiandewang Do you mean using threads in OCN does not speed up the OCN computation or it does not speed up the total run time for the test?